File: src/main/java/Analyse.java
import com.intellij.openapi.actionSystem.AnActionEvent;
import org.jetbrains.annotations.NotNull;

public class Analyse extends com.intellij.openapi.actionSystem.AnAction {

    @Override
    public void actionPerformed(@NotNull AnActionEvent e) {
        GitVcs.getAffectedFiles(e.getProject());
    }
}

File: src/main/kotlin/Analyse.kt
import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.ui.Messages

class Analyse : AnAction() {
    override fun actionPerformed(e: AnActionEvent) {
        Messages.showInputDialog("Insert Depth", "Analyse", Messages.getInformationIcon())
        AnalyseReport().showAndGet()
    }
}